aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -261,6 +267,71 @@ public static CreateDbInstanceRequest createDbInstanceRequest(
                 .build();
     }
 
+    static Instant stringToInstant(String restoreTimeString) {
+        if (com.amazonaws.util.StringUtils.hasValue(restoreTimeString)) {
+            try {
+                return LocalDateTime.parse(
+                                restoreTimeString,
+                                DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+                        )
+                        .atOffset(ZoneOffset.ofHours(0))
+                        .toInstant();
+            } catch (Exception e) {
+                throw new CfnInvalidRequestException(new Exception("Unable to parse RestoreTime"));
```

Comment:
This error would be visible to the customer. Firstly, we can use the instant format exception message. Secondly, we might use schema regex validator for this field. In this case it could be justified, given we're going to surface this cryptic parse error message anyways.

Reply:
Is my understanding correct:
2 options
- Throw the error as is. Which will be a cryptic regex
- Have a cfn level regex on the schema level which will throw a cryptic regex

Since cryptic regex is being thrown anyway we might as well do this from a schema level validation?